### PR TITLE
adding support for azure metrics

### PR DESCRIPTION
**Pull request by idohalevi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,16 +26,17 @@ Make sure to use these settings:
 * **Logzio Host**: Use your Logz.io region's listener URL.
   If your login URL is app.logz.io, use `listener.logz.io` (this is the default setting).
   If your login URL is app-eu.logz.io, use `listener-eu.logz.io`.
-* **Logzio Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.
+* **Logs Account Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the logs account you want to ship to.
+* **Metrics Account Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the metrics account you want to ship to.
```
Comment (idohalevi) [inline README.md]: @imnotashrimp 
The correct names are: Logzio Logs Host, Logzio Metrics Host, Logzio Logs Token, Logzio Metrics Token

File: README.md [hunk 2]
```diff
@@ -29,16 +26,17 @@ Make sure to use these settings:
...
 
-### 3. _(Optional)_ Add failsafe for log shipping timeouts
+### 3. _(Optional)_ Add failsafes for shipping timeouts
 
-You can configure logzio-azure-serverless to back up logs to Azure Blob Storage.
-So if the connection to Logz.io times out or an error occurs, you'll still have a backup of any dropped logs.
+You can configure logzio-azure-serverless to back up logs and metrics to Azure Blob Storage.
+So if the connection to Logz.io times out or an error occurs, you'll still have a backup of any dropped data that didn't get shipped.
 
 ![Function app left menu](img/function-app-menu-integrate.png)
```
Comment (idohalevi) [inline README.md]: @imnotashrimp 
The name of the functions changes so the image is not good anymore. For now, let's take in off(img/function-app-menu-integrate.png).

File: README.md (hunk 1 above)
Comment (imnotashrimp) [reply README.md]: What would metrics listener url be?

File: README.md (hunk 2 above)
Comment (imnotashrimp) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (idohalevi) [reply README.md]: @imnotashrimp 
Logzio Metrics Host - the options are the same as the logs host

**Pull request merged**